stdlib/3/asyncio/subprocess.pyi

```diff
@@ -39,22 +40,175 @@ class Process:
     def kill(self) -> None: ...
     async def communicate(self, input: Optional[bytes] = ...) -> Tuple[bytes, bytes]: ...
 
+@overload
+async def create_subprocess_shell(
+    cmd: Union[str, bytes],  # Union used instead of AnyStr due to mypy issue  #1236
```

Comment:
I'm not familiar with the issue, but it's closed now. Should I update these to `AnyStr`?

Reply:
`Union` actually looks correct to me.